Elsewhen/Extensions/TimeZone.swift

```diff
@@ -44,6 +48,12 @@ extension TimeZone {
                 return true
             }
         }
+        if let country = country {
+            let countryLower = country.lowercased()
+            if countryLower.contains(st) || countryLower.contains(st.replacingOccurrences(of: "_", with: " ")) {
```

Comment:
We could check for both strings with a single iteration [contains(where:)](https://developer.apple.com/documentation/swift/array/2297359-contains)

Reply:
My bad, I mis-parsed the types 🤦‍♂️